### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by sam9111**

### CODE REVIEW

**Comment by sam9111** [comment]
Weird A in the Heading

File: src/pages/index.js [hunk 1]
```diff
@@ -1,9 +1,277 @@
...
 import React from "react";
-import { Box, Button } from "@chakra-ui/react";
+import {
+  Container,
+  Stack,
+  Flex,
+  Box,
+  Heading,
+  Text,
+  Button,
```
Comment (harisarang) [inline src/pages/index.js:10]: Remove unused components

File: src/pages/index.js [hunk 2]
```diff
@@ -1,9 +1,277 @@
...
+        </Center>
+      </Flex>
+
+      <Stack
+        align={"center"}
+        spacing={{ base: 8, md: 10 }}
+        py={{ base: 20, md: 28 }}
+        direction={{ base: "column", md: "row" }}
+      >
+        <Flex
```
Comment (harisarang) [inline src/pages/index.js:148]: Since, this component is used more than once you seprate it as some card component

**Review by harisarang — CHANGES_REQUESTED** [review]
Other than these everything LGTM

**Pull request merged**